### PR TITLE
fix: output wraps in terminal

### PR DESCRIPTION
**Pull request by cafkafk**
Tracking-issue: https://github.com/cafkafk/eza/issues/103

### CODE REVIEW

**Comment by cafkafk** [comment]
@aarondill can you test this? 

Perhaps it would also need some comments explaining what it actually does and why?

File: src/output/grid.rs [hunk 1]
```diff
@@ -43,12 +44,23 @@ impl<'a> Render<'a> {
...
+
+            match filename.options.embed_hyperlinks {
+                EmbedHyperlinks::On => width = filename.bare_width(),
+                EmbedHyperlinks::Off => width = *contents.width(),
+            };
+
+            match filename.options.show_icons {
+                ShowIcons::On(_) => width += 1,
+                ShowIcons::Off => width += 0,
+            };
```
Comment (aarondill) [inline src/output/grid.rs:57]: If `EmbedHyperlinks::Off`, then we are using `*contents.width()`, which if I understand correctly *already includes the icon*. But if `ShowIcons::On(_)`, then we are adding an *additional* 1 to the width. 

Shouldn't we only add 1 when `EmbedHyperlinks::On`?

Comment (cafkafk) [reply src/output/grid.rs]: I think you're right!

Can you sanity check that the force push fixed this?

**Comment by aarondill** [comment]
Tested and it seems to work :smile: both with and without `--hyperlink` (though I can't test hyperlink functionality itself, that shouldn't have changed tho)

**Comment by cafkafk** [comment]
I've tested hyperlink functionality so I guess the issue is solved for now!

**Comment by aarondill** [comment]
Woo! This was my only blocker from using `eza` instead of `exa`! Time to `cargo install` :fireworks:

**Pull request merged**